.github/workflows/c3-e2e-quarantine.yml

```diff
@@ -12,7 +12,12 @@ env:
   bun-version: 1.0.3
 jobs:
   e2e:
+    timeout-minutes: 30
+    concurrency:
```

Comment:
is this here so that things don't run in parallel? (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency)

is this needed? is it so that we try not to get agents to hang for long period of time before picking up the jobs?

Reply:
It's intended to cancel jobs from previous commits in a PR when a new commit is pushed. It _shouldn't_ (I hope) affect concurrency of running jobs

Reply:
I see, yeah hopefully it works as you want it to, the github docs don't really explain it well in my opinion 😅